packages/web-service/src/pages/habitat/types/habitat-types.njk

```diff
@@ -0,0 +1,58 @@
+{% extends "standard-form.njk" %}
+
+{% from "error-summary.njk" import errorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set errorMap = {
+        'sett-selection': {
+          'no-radio-selected': { ref: '#sett-radio', text: 'You have not selected a sett type.' }
+        }
+    }
+%}
+
+{% block pageTitle %}{{ 'Error: ' + title if error else title }}{% endblock %}
+
+{% block pageContent %}
+
+{{ errorSummary(error, errorMap, 'There is a problem') }}
+
+  {{ govukRadios({
+    id: "sett-radio",
+    name: "sett-radio",
+    idPrefix: "where-do-you-live",
+    fieldset: {
+      legend: {
+        text: "What are the badgers using this sett for?",
+        isPageHeading: true,
+        classes: "govuk-fieldset__legend--l"
+      }
+    },
+    hint: {
+      text: "Select an option."
+    },
+    errorMessage: { text: "Select a sett type." } if error['sett-selection'],
+    items: [
+      {
+        value: "main",
```

Comment:
Bring these values down from the page -- there is a specific location for the keys, which are immutable 

`packages/powerapps-keys/keys/keys.js`

Reply:
Me and Victor will likely need to change these keys on Monday 

But Rich + Douglas aren't in today, so was hoping to merge this and just change the keys when I can clarify on Monday